src/main/scala/no/ndla/searchapi/service/search/SearchConverterService.scala

```diff
@@ -468,7 +469,8 @@ trait SearchConverterService {
         url = url,
         contexts = contexts,
         supportedLanguages = supportedLanguages,
-        learningResourceType = LearningResourceType.LearningPath.toString
+        learningResourceType = LearningResourceType.LearningPath.toString,
+        status = Some(api.Status(searchableLearningPath.status, Seq.empty))
```

Comment:
Du må sjekke at statusene for læringsstier eksisterer i ed. :
        PRIVATE: 'Kladd',
        PUBLISHED: 'Offentlig',
        UNLISTED: 'Lenkedelt',
        SUBMITTED: 'Bedt om offentliggjøring',

Reply:
i phrases tenker du?

Reply:
Ja, for at korrekte verdier skal kunne vises for læringsstier.